.hlint.yaml

```diff
@@ -1,17 +1,13 @@
-- arguments:
-  - -XTypeApplications # TypeApplications are disabled by default in HLint,
-                       # although the documentation says otherwise
+############################################################################
+## Universum
+############################################################################
```

Comment:
While we are at it, I think it would be good to add a comment stating that these rules are for people who use Universum as their prelude, not for people who develop Universum itself.